<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowserInfo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>231</width>
      <height>280</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="slot_1x1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(0, 255, 0); }
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="slot_3x3">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(255, 255, 255); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="slot_1x3">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(0, 255, 0); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="slot_2x2">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(255, 255, 255); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="slot_1x4">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(0, 255, 0); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="slot_4x2">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(255, 255, 255); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="slot_1x2">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(0, 255, 0); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="slot_3x2">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(255, 255, 255); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="slot_5x1">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(255, 0, 0); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="slot_5x2">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(255, 0, 0); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="slot_5x3">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(255, 0, 0); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="slot_5x4">
       <property name="styleSheet">
        <string notr="true">QPushButton {Height:50; Width : 50; border-radius: 25%; background-color: rgb(255, 0, 0); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonReset">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Info:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMoves">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No of moves:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdMoves">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 0, 0, 150);</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelET">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 0, 0, 150);
font: 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>  00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPause">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelElapsedTime">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Elapsed Time:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResume">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
